include/NovelRT/Transform.h

```diff
@@ -8,41 +8,74 @@
 #endif
 
 namespace NovelRT {
+  /**
+   * Represents a 2D position, scale and rotation.
+   */
   class Transform {
   protected:
     Maths::GeoVector2<float> _position;
     Maths::GeoVector2<float> _scale;
     float _rotation;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
+     * @param position The position, which represents the position of the transform in a given space such as screen or world space.
+     * @param rotation The rotation, in degrees, which represents the rotation of the transform.
+     * @param scale The scale, which represents the size of the transform.
+     */
     Transform(const Maths::GeoVector2<float>& position, float rotation, const Maths::GeoVector2<float>& scale) noexcept;
+    /**
+     * Creates a new Transform with the default values.
+     */
     Transform() noexcept;
 
+    /**
+     * Gets the Axis Aligned Bounding Box based on the position and scale of this transform.
+     */
     inline Maths::GeoBounds getAABB() const {
       auto scale = fmaxf(_scale.getX(), _scale.getY());
       return Maths::GeoBounds(_position, Maths::GeoVector2(scale, scale), 0);
     }
 
+    /**
+     * Gets the bounds, based on the position, scale and rotation of this transform.
+     */
     inline Maths::GeoBounds getBounds() const {
       return Maths::GeoBounds(_position, _scale, _rotation);
     }
 
+    /**
+     * Gets the position of this transform.
```

Comment:
Maybe add to this, `This is the read-only version.`? Or something to indicate this is the version that gets used in `const` contexts. :D

Reply:
Ah yeah, my previous attempt was with "constant" 😅